test/slow/nvm reinstall-packages/should work as expected

```diff
@@ -6,16 +6,17 @@ die () { echo "$@" ; exit 1; }
 
 nvm use 0.10.28
 
-EXPECTED_PACKAGES="autoprefixer bower david eslint grunt-cli grunth-cli http-server jshint marked node-gyp recursive-blame uglify-js yo"
+EXPECTED_PACKAGES="autoprefixer bower david eslint grunt-cli grunth-cli http-server jshint marked node-gyp npmlist recursive-blame uglify-js yo"
```

Comment:
Does `npmlist` require `grunt-cli`, which requires a new version of node? The tests run with `0.10` at the moment. Could we use another package that doesn't have those kind of requirements?